simphony_ui/ui.py

```diff
@@ -27,6 +29,34 @@ def dataset2cudssource(dataset):
     return CUDSSource(cuds=dataset)
 
 
+class CustomErrorMessage(HasStrictTraits):
+
+    # The message to be shown:
+    message = Str('Oups ! Something went wrong...')
+
+    # The time (in seconds) to show the message:
+    time = Float(20)
+
+    def show(self):
+        """ Display the error message for a limited duration.
+        """
+        view = View(
+            HGroup(
+                spring,
+                Item('message',
+                     show_label=False,
+                     style='readonly'
+                     ),
+                spring
+            ),
+            buttons=['OK'],
+            title='Error'
+        )
+
+        do_after(int(1000.0 * self.time),
+                 self.edit_traits(view=view).dispose)
```

Comment:
You will have to mock the dialog in the tests, so that it doesn't show up. if you use `.error` it's just a matter of patching a function as we usually do (and then check if it was called)